app/assets/stylesheets/patterns/_card.scss

```diff
@@ -2,10 +2,11 @@
 
 // Base class
 .card {
+  @include word-wrap();
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets